models/user/user.go

```diff
@@ -872,16 +908,21 @@ func UpdateUser(u *User, emailChanged bool) error {
 }
 
 // UpdateUserCols update user according special columns
-func UpdateUserCols(ctx context.Context, u *User, cols ...string) error {
-	return updateUserCols(db.GetEngine(ctx), u, cols...)
+func UpdateUserCols(ctx context.Context, u *User, force bool, cols ...string) error {
```

Comment:
I'd mostly see that force = false, I think it's better if the exported function doesn't have this as a argument but rather a new function `UpdateForceUserCols`, the unexported function is fine to have such argument IMO

Reply:
Done.